MLS.Agent.Tests/ApiViaHttpTests.cs

```diff
@@ -1018,6 +1018,22 @@ public async Task Scaffolding_HTML_trydotnet_js_autoEnable_useWasmRunner_is_true
                        .Contain(s => s.Contains(@"trydotnet.autoEnable({ apiBaseAddress: new URL(""http://localhost""), useWasmRunner: true });"));
             }
         }
+
+        [Fact]
+        public async Task Is_able_to_serve_static_files()
+        {
+            var options = new StartupOptions(dir: TestAssets.SampleConsole);
+
+            using (var agent = new AgentService(options: options))
+            {
+                var response = await agent.GetAsync(@"/a.html");
+
+                response.Should().BeSuccessful();
+                var html = await response.Content.ReadAsStringAsync();
+                html.Should().Be("<p>This file should be served</p>");
```

Comment:
Since the Markdown parser will pass HTML through, this test doesn't actually ensure that the static file was served. If the Markdown route matches this URL, you'll see the same result.